etc/nginx/nginx.conf.ctmpl

```diff
@@ -61,4 +61,4 @@ http {
         }
         {{end}}
     }
-}
+}
```

Comment:
Does your editor remove trailing newlines on save? I notice this pop up sometimes with your PRs. It probably doesn't matter in this case but this can mess with some parsers and tools ([POSIX standard](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) says a line has to end with a newline so naturally some tools are pedantic about it) so I have mine configured to make sure the newline is there.

(I also have it trim off trailing whitespace but that's because I worked in a whitespace-sensitive language a lot.)